README.md

```diff
@@ -121,12 +121,18 @@ done
 
 ### Configure Helm chart values
 
-Karpenter Helm chart requires some configuration via values to work with a specific AKS cluster. The values are documented in the Helm chart itself, but you can use `configure-values.sh` to generate `karpenter-values.yaml` with the required configuration. The script interrogates the AKS cluster and generates the values file, using `karpenter-values-template.yaml` as a template. (The script fetches the template automatically.)
+The Karpenter Helm chart requires specific configuration values to work with an AKS cluster. While these values are documented within the Helm chart, you can use the `configure-values.sh` script to generate the `karpenter-values.yaml` file with the necessary configuration. This script queries the AKS cluster and creates the values file using `karpenter-values-template.yaml` as a template. Although the script automatically fetches the template from the main branch, inconsistencies may arise between the installed version of Karpenter and the repository code. Therefore, it is advisable to download the specific version of the template before running the script.
 
 ```bash
+# Select version to install
+export KARPENTER_VERSION=0.5.1
```

Comment:
why 0.5.1?

Reply:
Because it was the version already present in the readme. Should i update it to the latest?